.circleci/config.yml

```diff
@@ -177,6 +177,12 @@ jobs:
             curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
       - attach_workspace:
           at: ~/uport-mobile
+      - run:
+            name: Install appium server
+            command: |
+              npm install -g appium
+              npm install wd
```

Comment:
Is this a global package or local? If local, why isn't it in `package.json`

Reply:
I’m going to check if I can add installation of appium server to package.json

As for running tests locally, you have to install appium:
npm install -g appium
npm install wd
pip install Appium-Python-Client

Then launch appium in terminal by running “appium”
And in a separate terminal run the tests from repo “python testsuites/test.py”